Clojure-Conditionals.md

```diff
@@ -0,0 +1,103 @@
+# Clojure - Conditionals
+
+You're not going to get anywhere in a language if all you can do is define
+functions that print things and do simple arithmetic. Conditionals and logic
+are a fundamental part of making code that does interesting, useful things. Try
+and imagine a world without logic in programs: you wouldn't even be able to simple
+things, like checking if two numbers are equal!
+
+### Logical Operators
```

Comment:
Try to keep the heading in ascending order if possible.

After `#` all other top level headings should be `##` and bellow those, the top ones should be `###` and so on.